### PR TITLE
[Only for review] v2.5 release notes

### PR DESCRIPTION
**Pull request by chensuyue**
## Type of Change

v2.5 release notes, for review only.

### CODE REVIEW

File: v2.5_release_notes.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- Highlights
+- Features
+- Improvement
+- Productivity
+- Bug Fixes
+- External Contributes
+- Validated Configurations
+
+**Highlights**
+- Integrated Weight-Only Quantization algorithm [AutoRound](https://github.com/intel/auto-round) and verified on Gaudi2, Intel CPU, NV GPU
```
Comment (hshen14) [inline v2.5_release_notes.md:12]: How about Intel GPU?

Comment (chensuyue) [reply v2.5_release_notes.md:12]: It's not supported.

File: v2.5_release_notes.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+
+**Features**
+- [Quantization] Integrate Weight-Only Quantization algorithm [AutoRound](https://github.com/intel/auto-round) ([5c7f33](https://github.com/intel/neural-compressor/commit/5c7f336037e602321efec71aa516bc4fade082c8), [dfd083](https://github.com/intel/neural-compressor/commit/dfd083df3234aa27548391c381d3ac9dc8139676), [9a7ddd](https://github.com/intel/neural-compressor/commit/9a7ddda6f852792663e9cf3f076b88caa32e83a8), [cf1de7](https://github.com/intel/neural-compressor/commit/cf1de74608c4dc16bb18abee8f9680985af5ad6e))
+- [Quantization] Quantize weight with in-place mode in Weight-Only Quantization ([deb1ed](https://github.com/intel/neural-compressor/commit/deb1ed51cd2ad9768318ae49763898d3bb7af663))
+- [Pruning] Enable SNIP on multiple cards using DeepSpeed ZeRO-3 ([49ab28](https://github.com/intel/neural-compressor/commit/49ab28d362b03338194160e5ef67a3b8c7967a86))
+- [Pruning] Support new pruning approach Wanda for PyTorch LLM ([7a3671](https://github.com/intel/neural-compressor/commit/7a367179804565777241f73a87f903c82b6723e0))
```
Comment (thuang6) [inline v2.5_release_notes.md:20]: in fact, this PR added two algorithms:  WANDA and DSNOT

Comment (chensuyue) [reply v2.5_release_notes.md]: updated

**Pull request closed without merging**